src/PKSim.Core/Services/DataRepositoryFromResultsCreator.cs

```diff
@@ -41,11 +54,32 @@ public DataRepository CreateResultsFor(IndividualSimulation simulation)
 
          var individualResults = simulation.Results.First();
          individualResults.Select(value => dataColumnFor(value, simulation, timeColumn))
-            .Where(col=> col!=null).Each(dataRepository.Add);
+            .Where(col => col != null).Each(dataRepository.Add);
 
+         UpdateColumnInternalUse(simulation, dataRepository);
          return dataRepository;
       }
 
+      public void UpdateColumnInternalUse(IndividualSimulation simulation, DataRepository dataRepository = null)
```

Comment:
@Yuri05 This function was private. I moved it and made it public. The issue was that when we load from project, we always recreate a DataRepository from scratch. There, the knowledge of intermal columns was lost